File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/FormScreenInfo.java
package nta.med.data.model.ihis.system;

public class FormScreenInfo {
	private String sysId;
    private String pgmNm;
    private Double pgmEntGrad;
    private Double pgmUpdGrad;
    private String pgmScrt;
    private String pgmDupYn;
    private String asmName;
    private String asmPath;
    private String asmVer;
    private String grpId;
	public FormScreenInfo(String sysId, String pgmNm, Double pgmEntGrad,
			Double pgmUpdGrad, String pgmScrt, String pgmDupYn, String asmName,
			String asmPath, String asmVer, String grpId) {
		super();
		this.sysId = sysId;
		this.pgmNm = pgmNm;
		this.pgmEntGrad = pgmEntGrad;
		this.pgmUpdGrad = pgmUpdGrad;
		this.pgmScrt = pgmScrt;
		this.pgmDupYn = pgmDupYn;
		this.asmName = asmName;
		this.asmPath = asmPath;
		this.asmVer = asmVer;
		this.grpId = grpId;
	}
	public String getSysId() {
		return sysId;
	}
	public void setSysId(String sysId) {
		this.sysId = sysId;
	}
	public String getPgmNm() {
		return pgmNm;
	}
	public void setPgmNm(String pgmNm) {
		this.pgmNm = pgmNm;
	}
	public Double getPgmEntGrad() {
		return pgmEntGrad;
	}
	public void setPgmEntGrad(Double pgmEntGrad) {
		this.pgmEntGrad = pgmEntGrad;
	}
	public Double getPgmUpdGrad() {
		return pgmUpdGrad;
	}
	public void setPgmUpdGrad(Double pgmUpdGrad) {
		this.pgmUpdGrad = pgmUpdGrad;
	}
	public String getPgmScrt() {
		return pgmScrt;
	}
	public void setPgmScrt(String pgmScrt) {
		this.pgmScrt = pgmScrt;
	}
	public String getPgmDupYn() {
		return pgmDupYn;
	}
	public void setPgmDupYn(String pgmDupYn) {
		this.pgmDupYn = pgmDupYn;
	}
	public String getAsmName() {
		return asmName;
	}
	public void setAsmName(String asmName) {
		this.asmName = asmName;
	}
	public String getAsmPath() {
		return asmPath;
	}
	public void setAsmPath(String asmPath) {
		this.asmPath = asmPath;
	}
	public String getAsmVer() {
		return asmVer;
	}
	public void setAsmVer(String asmVer) {
		this.asmVer = asmVer;
	}
	public String getGrpId() {
		return grpId;
	}
	public void setGrpId(String grpId) {
		this.grpId = grpId;
	}
    
	}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/FormScreenInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
	public class FormScreenInfo
	{
		private String _sysId;
		private String _pgmNm;
		private String _pgmEntGrad;
		private String _pgmUpdGrad;
		private String _pgmScrt;
		private String _pgmDupYn;
		private String _asmName;
		private String _asmPath;
		private String _asmVer;
		private String _grpId;

		public String SysId
		{
			get { return this._sysId; }
			set { this._sysId = value; }
		}

		public String PgmNm
		{
			get { return this._pgmNm; }
			set { this._pgmNm = value; }
		}

		public String PgmEntGrad
		{
			get { return this._pgmEntGrad; }
			set { this._pgmEntGrad = value; }
		}

		public String PgmUpdGrad
		{
			get { return this._pgmUpdGrad; }
			set { this._pgmUpdGrad = value; }
		}

		public String PgmScrt
		{
			get { return this._pgmScrt; }
			set { this._pgmScrt = value; }
		}

		public String PgmDupYn
		{
			get { return this._pgmDupYn; }
			set { this._pgmDupYn = value; }
		}

		public String AsmName
		{
			get { return this._asmName; }
			set { this._asmName = value; }
		}

		public String AsmPath
		{
			get { return this._asmPath; }
			set { this._asmPath = value; }
		}

		public String AsmVer
		{
			get { return this._asmVer; }
			set { this._asmVer = value; }
		}

		public String GrpId
		{
			get { return this._grpId; }
			set { this._grpId = value; }
		}

		public FormScreenInfo() { }

		public FormScreenInfo(String sysId, String pgmNm, String pgmEntGrad, String pgmUpdGrad, String pgmScrt, String pgmDupYn, String asmName, String asmPath, String asmVer, String grpId)
		{
			this._sysId = sysId;
			this._pgmNm = pgmNm;
			this._pgmEntGrad = pgmEntGrad;
			this._pgmUpdGrad = pgmUpdGrad;
			this._pgmScrt = pgmScrt;
			this._pgmDupYn = pgmDupYn;
			this._asmName = asmName;
			this._asmPath = asmPath;
			this._asmVer = asmVer;
			this._grpId = grpId;
		}

	}
}